connect-examples/v2/node_payment/views/index.jade

```diff
@@ -1,310 +1,47 @@
-
-            lineItems: [
-              {
-                label: "Subtotal",
-                amount: "1.00",
-                pending: false,
-              },
-              {
-                label: "Tax",
-                amount: "0.01",
-                pending: false,
-              }
-            ]
-          };
-        },
-      }
-    });
-
-    // build payment form after DOM load
-    document.addEventListener('page:change', function(){
-      console.log('dom loaded')
-      paymentForm.build()
-    });
-
-    var paymentFormSubmit = function(){
-      console.log('submit clicked');
-      document.getElementById('submit').disabled = true;
-      paymentForm.requestCardNonce();
-      return false;
-    }
-
-    var chargeCardWithNonce = function(nonce) {
-      var product_id = document.getElementById('product_id').value;
-      var name = document.getElementById('name').value;
-      var email = document.getElementById('email').value;
-      var street_address_1 = document.getElementById('street_address_1').value;
-      var street_address_2 = document.getElementById('street_address_2').value;
-      var city = document.getElementById('city').value;
-      var state = document.getElementById('state').value;
-      var zip = document.getElementById('zip').value;
-      
-      var http = new XMLHttpRequest();
-      var url = "/charges/charge_card";
-      var params = "location_id=" + '#{square_location_id}'
-      + "&product_id=" + product_id 
-      + "&name=" + name 
-      + "&email=" + email 
-      + "&nonce=" + nonce
-      + "&street_address_1=" + street_address_1
-      + "&street_address_2=" + street_address_2
-      + "&city=" + city
-      + "&state=" + state
-      + "&zip=" + zip;
-
-      http.open("POST", url, true);
-
-      //Send the proper header information along with the request
-      http.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
-      http.setRequestHeader("X-CSRF-Token", "<%= form_authenticity_token %>");
-
-      http.onreadystatechange = function() {//Call a function when the state changes.
-          if(http.readyState == 4 && http.status == 200) {
-            var data = JSON.parse(http.responseText)
-            if (data.status == 200) {
-              document.getElementById("successNotification").style.display = "block";
-              document.getElementById("payment-form").style.display = "none";
-              window.scrollTo(0, 0);
-            }else if (data.status == 400){
-              var error_html = ""
-              for (var i =0; i < data.errors.length; i++){
-                error_html += "<li> " + data.errors[i].detail + " </li>";
-              }
-              document.getElementById("card-errors").innerHTML = error_html;
-              document.getElementById('submit').disabled = false;
-            }
-          }
-      }
-      http.send(params);
-    }
+  script(type="text/javascript" src="https://js.squareup.com/v2/paymentform")
```

Comment:
I was just checking what Jade was, and it seems like Jade package is outdated according tohttps://www.npmjs.com/package/jade. Should we convert to `pug`?